Sources/Plasma/Apps/plClient/winmain.cpp

```diff
@@ -972,6 +975,25 @@ LONG WINAPI plCustomUnhandledExceptionFilter( struct _EXCEPTION_POINTERS *Except
 #endif // HS_DEBUGGING
 }
 
+uint32_t ParseRendererArgument(const ST::string& requested)
+{
+    using namespace ST::literals;
+
+    if (requested.compare_i("directx"_st) == 0
+            || requested.compare_i("direct3d"_st) == 0
+            || requested.compare_i("dx"_st) == 0
+            || requested.compare_i("d3d"_st) == 0) {
```

Comment:
Part of me wonders if it would be better to check against a set of values (or use a regex) for cleaner code... eg
```C++
static std::unordered_map<ST::string, ST::hash_i, ST::equal_i> dx{
    "directx"_st, "direct3d"_st, "dx"_st, "d3d"_st
};
if (dx.find(requested) != dx.end())
    return hsG3DDeviceSelector::kDevTypeDirect3D;
```

Reply:
Sounds like a good plan to me